README.md

```diff
@@ -61,9 +68,5 @@ yarn run lint:fix
 yarn test
 ```
 
-## What's next
-
-Please check out [LoopBack 4 documentation](https://loopback.io/doc/en/lb4/) to
-understand how you can continue to add features to this application.
-
-[![LoopBack](https://github.com/strongloop/loopback-next/raw/master/docs/site/imgs/branding/Powered-by-LoopBack-Badge-(blue)-@2x.png)](http://loopback.io/)
+## [Contribution guidelines](https://github.com/pesto-students/babitz-backend/blob/Contribution.md/Contributing.md)
+If you wish to contribute, please go through the [contribution guidelines](https://github.com/pesto-students/babitz-backend/blob/Contribution.md/Contributing.md). 
```

Comment:
This should point to the main branch. You can either add this section in the Contribution guidelines PR or merge that one first and update the link here.